### PR TITLE
Added RxSwift conventions

### PR DESCRIPTION
**Pull request by RuudPuts**

### CODE REVIEW

File: swift/rx_swift.md [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+- [ViewControllers](#viewcontrollers)
+   + [Binding to ReactiveConnectable](#binding-to-reactiveconnectable)
+
+## Protocols
+
+In order to assure all (new) MVVM powered views are implemented in the same way 
+
+### ReactiveConnectable
+
+`ReactiveConnectable` is thé base protocol for any `-Model` type class, for example a `ViewModel` or a `TrackingModel`. It requires any implementation to define an `Input` & `Output` type, most often implemented in teh form of a struct.
```
Comment (MelleI) [inline swift/rx_swift.md]: typo: `teh` > `the`

File: swift/rx_swift.md [hunk 2]
```diff
@@ -0,0 +1,145 @@
+# RxSwift conventions
+
+This document contains all conventions decided upon by our team on how to implement RxSwift in our MVVM architecture.
+
+Other then these conventions a number of convenience extensions can be found in `ReactiveExtensions.swift` within the `JumboCore` module.
```
[the remark this answers is not in the document]
Comment (RuudPuts) [reply swift/rx_swift.md]: I find it important to note this file here, at least it's existence. Don't really mind wether or not someone public reads this line, the first goal of this file is a reference for our team.

File: swift/rx_swift.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (RuudPuts) [reply swift/rx_swift.md]: That's why it says 'most often', which I'll change to 'most of the times'

File: swift/rx_swift.md [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+
+    func bind(to output: ExampleViewModel.Output) {
+        ...
+    }
+}
+```
+
+As soon as analytics would be added to this view controller simply add another extension
+
+```
```
[the remark this answers is not in the document]
Comment (RuudPuts) [reply swift/rx_swift.md]: Didn't want to make it too bloated, but fine

**Pull request merged**